js/exercise-03.js

```diff
@@ -0,0 +1,14 @@
+/* 
+    1. Define function 'getTypeOf', which receives a value as paramenter and returns the type of this value
+        Use typeof to ckech the types for each of the main 5 primitive types (string, number, boolean, null, undefined)
+    2. Log the result of any function call.
+*/
+
+function getTypeOf(value) {
+  return typeof value;
+}
+console.log(getTypeOf("Paco"));
```

Comment:
Adding a comment at each invocation with expected output is always useful

```suggestion
console.log(getTypeOf("Paco")); //string
```
⚠️ FIX: add execution output for each log